src/applications/disability-benefits/all-claims/utils/on-submit/purge-toxic-exposure-data.js

```diff
@@ -216,14 +238,6 @@ export const purgeToxicExposureData = formData => {
   });
   toxicExposure = processedExposure;
 
-  // Remove unchecked conditions
```

Comment:
Decided to keep false conditions and other exposures so BE logging knows what `should` be removed so we can implicitly infer purging on BE.

Implemented in this BE PR.

Reply:
This answers my question from the BE PR which I looked at first. 

I think this could be in either place, FE or BE, or both. Not positive which makes more sense. I originally thought in the FE transform because that is where we do other similar logic and transforms, but we transform stuff on the BE too, and like you said that has the advantage of being able to log better/easier.